geode-wan/src/test/java/org/apache/geode/internal/cache/wan/misc/WanAutoDiscoveryDUnitTest.java

```diff
@@ -293,43 +293,50 @@ public void test_NY_Recognises_TK_AND_HK_Simultaneously() {
 
   @Test
   public void test_LN_Sender_recognises_ALL_NY_Locators() {
+    IgnoredException ie = IgnoredException
+        .addIgnoredException("could not get remote locator information for remote site");
+    try {
+      Integer lnLocPort1 = (Integer) vm0.invoke(() -> WANTestBase.createFirstLocatorWithDSId(1));
 
-    Integer lnLocPort1 = (Integer) vm0.invoke(() -> WANTestBase.createFirstLocatorWithDSId(1));
-
-    Integer lnLocPort2 = (Integer) vm5.invoke(() -> WANTestBase.createSecondLocator(1, lnLocPort1));
-
-    vm2.invoke(() -> WANTestBase.createCache(lnLocPort1, lnLocPort2));
+      Integer lnLocPort2 =
+          (Integer) vm5.invoke(() -> WANTestBase.createSecondLocator(1, lnLocPort1));
 
-    vm2.invoke(() -> WANTestBase.createSender("ln", 2, false, 100, 10, false, false, null, true));
+      vm2.invoke(() -> WANTestBase.createCache(lnLocPort1, lnLocPort2));
 
-    Integer nyLocPort1 =
-        (Integer) vm1.invoke(() -> WANTestBase.createFirstRemoteLocator(2, lnLocPort1));
+      vm2.invoke(() -> WANTestBase.createSender("ln", 2, false, 100, 10, false, false, null, true));
 
-    vm2.invoke(() -> WANTestBase.startSender("ln"));
+      Integer nyLocPort1 =
+          (Integer) vm1.invoke(() -> WANTestBase.createFirstRemoteLocator(2, lnLocPort1));
 
-    // Since to fix Bug#46289, we have moved call to initProxy in getConnection which will be called
-    // only when batch is getting dispatched.
-    // So for locator discovery callback to work, its now expected that atleast try to send a batch
-    // so that proxy will be initialized
-    vm2.invoke(
-        () -> WANTestBase.createReplicatedRegion(getTestMethodName() + "_RR", "ln", isOffHeap()));
+      vm2.invoke(() -> WANTestBase.startSender("ln"));
 
-    vm2.invoke(() -> WANTestBase.doPuts(getTestMethodName() + "_RR", 10));
+      // Since to fix Bug#46289, we have moved call to initProxy in getConnection which will be
```

Comment:
Is this a newly added comment?  If so, I don't think there is a GEODE ticket 46289...

Reply:
Nvm just noticed it was an old moved comment...